orchestration/dagster_orchestration/hca_orchestration/solids/stage_data.py

```diff
@@ -57,3 +59,36 @@ def pre_process_metadata(context: AbstractComputeExecutionContext) -> Nothing:
         input_prefix=context.solid_config["input_prefix"],
         output_prefix=f'gs://{bucket_name}/{prefix_name}'
     )
+
+
+@solid(
+    required_resource_keys={"bigquery_client", "load_tag"},
+    config_schema={
+        "staging_bq_project": String,
+        "staging_dataset_prefix": String,
+        "staging_table_expiration_ms": Int
+    },
+    input_defs=[InputDefinition("start", Nothing)],
+    output_defs=[OutputDefinition(name="staging_dataset_name", dagster_type=String)]
+)
+def create_staging_dataset(context: AbstractComputeExecutionContext) -> String:
```

Comment:
nit: this should return a `str` rather than a `String`  - `String`  here is the dagster config property data type, but we're returning a bare python string